dataprofiler/tests/labelers/test_column_name_model.py

```diff
@@ -113,51 +94,22 @@ def test_help(self, mock_stdout):
 
     @mock.patch("sys.stdout", new_callable=StringIO)
     def test_predict(self, mock_stdout):
-        parameters = {
-            "true_positive_dict": [
-                {"attribute": "ssn", "label": "ssn"},
-                {"attribute": "suffix", "label": "name"},
-                {"attribute": "my_home_address", "label": "address"},
-            ],
-            "false_positive_dict": [
-                {
-                    "attribute": "contract_number",
-                    "label": "ssn",
-                },
-                {
-                    "attribute": "role",
-                    "label": "name",
-                },
-                {
-                    "attribute": "send_address",
-                    "label": "address",
-                },
-            ],
-        }
-        model = ColumnNameModel(parameters=parameters)
-
-        expected_output = {
-            "ssn": {
-                "pred": "ssn",
-            }
-        }
-
+        # test show confidences
+        model = ColumnNameModel(parameters=mock_model_parameters)
+        expected_output = [[100.0, 0]]
         with self.assertLogs(
             "DataProfiler.labelers.column_name_model", level="INFO"
         ) as logs:
             model_output = model.predict(data=["ssn", "role_name", "wallet_address"])
-
-        self.assertIn("pred", model_output["ssn"])
         self.assertTrue(np.array_equal(expected_output, model_output))
-
         self.assertTrue(len(logs.output))
 
-        # test show confidences
-        expected_output = {"ssn": {"pred": "ssn", "conf": 100.0}}
-        model_output = model.predict(
-            data=["ssn", "role_name", "wallet_address"], show_confidences=True
-        )
-        self.assertTrue(np.array_equal(expected_output, model_output))
+        # `show_confidences` is disabled currently
+        # should raise error if set to `True`
+        with self.assertRaises(NotImplementedError):
+            model.predict(
+                data=["ssn", "role_name", "wallet_address"], show_confidences=True
+            )
```

Comment:
disabled `show_confidences` right now. Need to clean this up and enable it as a toggle

Reply:
an option is to put this in the post processor part of the labeler, but that breaks the API paradigm for the `model's` `predict()` method ...